nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -341,27 +349,31 @@ def _plot_surf(
         cbar_vmax=None,
         cbar_tick_format="auto",
         title=None,
-        title_font_size=18,
+        title_font_size=None,
```

Comment:
Why do you change this ?

Reply:
This is only used for "plotly". As I've put checks for parameters not implemented in "matplotlib" which raises warning if the value is different from `None`, I set the value to None by default. 
In `_plotly_backend` if the value is `None`, it is set to `18`.

Reply:
I've done the same for `symmetric_cmap` which also is not used for "matplotlib".